.github/workflows/api-ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         cd torch_ort
         python3 -m pip install --upgrade protobuf==3.20.1
         python3 -m pip install --pre onnxruntime-training --no-cache-dir -f https://download.onnxruntime.ai/onnxruntime_nightly_cpu.html
-        python3 -m pip install torch --extra-index-url https://download.pytorch.org/whl/cpu
+        python3 -m pip install torch -f https://download.pytorch.org/whl/cpu
```

Comment:
This will still install the cuda torch package. Surprisingly, the build still passes.

I think to be safer; we need to install like so: `python3 -m pip install torch==1.13.0+cpu -f https://download.pytorch.org/whl/torch_stable.html`